backend/src/main/java/com/dbms/georgia_express/model/Card.java

```diff
@@ -0,0 +1,173 @@
+package com.dbms.georgia_express.model;
+
+import jakarta.persistence.*;
+
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "card")
+public class Card {
+
+    @Id
+    @Column(name = "cardNumber")
```

Comment:
This will still break. Change this to card_number and create a new flyway file to modify the column names. Example: https://github.com/Hellblazer1804/Georgia-Express/pull/6/files#diff-3394d9b047b10064783a4f4f722730c9700580bde9a7dc6b3828258a3bac214c